packages/@lightningjs/ui-components/src/components/ListItem/ListItem.stories.js

```diff
@@ -24,8 +24,21 @@ import { default as Checkbox } from '../Checkbox';
 import { default as Radio } from '../Radio';
 import { default as Toggle } from '../Toggle';
 
+/**
+ * ListItem component with the ability to let a user pick from a list of options.
+ */
 export default {
-  title: 'Components/ListItem/ListItem'
+  title: 'Components/ListItem/ListItem',
+  args: {
```

Comment:
not a necessary change but just exploring some more code clean up and looking to CSF3 format
[Storybook Args](https://storybook.js.org/docs/html/writing-stories/args)